tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivativeImpl.tpp

```diff
@@ -1014,30 +1030,25 @@ void test_impl(const Spectral::Quadrature quadrature,
                 system_type == SystemType::Mixed) {
     for (size_t i = 0; i < mesh.number_of_grid_points(); i += 2) {
       if (quadrature == Spectral::Quadrature::GaussLobatto) {
-        get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i] = 25.5;
-        get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i + 1] = 37.5;
+        get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i] = 26.0;
+        get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i + 1] = 39.0;
       } else if (quadrature == Spectral::Quadrature::Gauss) {
         get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i] =
-            25.8660254037844375;
+            26.7320508075688785;
         get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i + 1] =
-            38.598076211353316;
+            41.196152422706632;
```

Comment:
Because I have no idea where these numbers came from, it is difficult to review changes to them